elastic/connectors-ruby
# Functional test

## Closes https://github.com/elastic/enterprise-search-team/issues/2386

Adds `make ftest`

That target will:

- create a stack in Docker (Mongo+Elasticsearch)
- add sample data in mongo (airbnb)
- run a full sync
- verify that Elasticsearch received the documents
- tear down the stack


in a follow-up issue we'll have it running once a day in Jenkins
